### ISSUE TITLE
[bw_related] shortcode not expanded on the Course tab 

### ISSUE THREAD

**Issue by bobbingwide** [bobbingwide/tags#14]
When viewing the latest event yesterday I noticed that the Course tab displayed the details for the course but the `[bw_related]` shortcode from that post's content was not expanded.

https://www.anchorgolf.com/event/25-jun-2022-the-hampshire/?oik-tab=course

### Explanation

- The shortcode is part of a Reusable block called **Event history**.

![image](https://user-images.githubusercontent.com/2474435/175805450-03d24781-235a-4b50-9107-d478e52a619a.png)

- The shortcode is expanded when we view the Course directly.
- But it's not expanded when the reusable block is embedded in content from `[bw_pages]`

The problem also occurs in the development environment.

**Comment by bobbingwide** [comment]
A possible workaround is to change the `[bw_related]` shortcode to the Dynamic Content Block displaying the equivalent. 
See prototype in https://s.b/tags/course/chichester-tower/
Additional parameters are needed to show all the related items. 

... but this doesn't work in the Course tab for the Events page either.

**Comment by bobbingwide** [comment]
The prototype was to use the Reusable block
```
<!-- wp:block {"ref":16263} /-->
```
which contained
```
<!-- wp:heading -->
<h2>Events</h2>
<!-- /wp:heading -->

<!-- wp:paragraph -->
<p>[bw_related post_type=event meta_key=_course orderby=_date order=desc]</p>
<!-- /wp:paragraph -->

<!-- wp:paragraph -->
<p></p>
<!-- /wp:paragraph -->

<!-- wp:oik/content-block {"shortcode":"bw_related","parameters":"meta_key=_course ","post_type":"event","format":"L"} /-->
```

**Comment by bobbingwide** [comment]
Instead of using the prototype to preferred soution is to 
- automatically add the `[bw_related]` shortcode by adding logic to `tags_the_post_course()`
- automatically add the `oik-block/fields` block as well
- call `tags_the_post_course()` from `TAGS_event_content::course()`

**Comment by bobbingwide** [comment]
Fix delivered in v0.4.0.
There are some entries which need to be manually updated to remove shortcodes and blocks that prevent the solution from working.

There also appears to be a glitch where the Events list shown on the Course tab doesn't include the current event. 
This can probably be resolved by using the `exclude=-1` attribute.

**Comment by bobbingwide** [comment]
Improved fix delivered in v0.4.1
